package.json

```diff
@@ -21,6 +21,7 @@
     "@prisma/debug": "^5.3.1",
     "@prisma/generator-helper": "^5.3.1",
     "dotenv": "^16.3.1",
+    "throat": "^6.0.2",
```

Comment:
This can be moved to `devDependencies`. This script will only ever be run from a developer's machine manually, and isn't needed for end users.

Reply:
Ok done!